PhysicsTools/NanoAOD/python/custom_btv_cff.py

```diff
@@ -684,8 +694,8 @@ def addPFCands(process, allPF = False, addAK4=False, addAK8=False):
 btvNano_switch = cms.PSet(
     btvNano_addAK4_switch = cms.untracked.bool(True),
     btvNano_addAK8_switch = cms.untracked.bool(False),
-    btvNano_addallPF_switch = cms.untracked.bool(False),
-    TaggerInput = cms.string("btvSF")
+    btvNano_addallPF_switch = cms.untracked.bool(True),
+    TaggerInput = cms.string("RobustParTAK4")
```

Comment:
Hi @lyazj thank you for the changes! I notice the changes here need to be reverted. Since we only want to partial tagger inputs which configured in `btvSF` and also not storing all PF cands.
Can you please change it?
Thanks